src/features/content-explorer/item-list/ItemList.js

```diff
@@ -210,6 +210,15 @@ const ItemList = ({
         };
     }
 
+    const rowCount = items.length;
+    const headerProps =
+        rowCount > 0
+            ? {
+                  headerHeight,
+                  headerRowRenderer: headerRenderer,
+              }
+            : {};
```

Comment:
maybe this could utilize the `tableProps`:
```suggestion
    if (items.length > 0) {
        tableProps.headerHeight = headerHeight;
        tableProps.headerRowRenderer = headerRenderer;
    }
```